core/persistence-api/src/main/java/org/apache/syncope/core/persistence/api/entity/auth/SAML2SP.java

```diff
@@ -73,4 +75,28 @@
     void setServiceProviderNameIdQualifier(String location);
 
     String getServiceProviderNameIdQualifier();
+
```

Comment:
I wonder if we could use an existing Enum or create new one to use instead of plain strings.

Reply:
Looks around. Can't find an existing enum, but it would be feasible to create an enum of all algorithms listed here:
https://www.w3.org/TR/xmlsec-algorithms/

Apache has a component that lists everything as a String constant:
https://santuario.apache.org/Java/api/org/apache/xml/security/encryption/XMLCipher.html

Perhaps this can be used in the enum we'd create? This seems like it would be consistent with the approach we take elsewhere in WA.

Reply:
I agree: FYI the given class is available at https://github.com/apache/santuario-xml-security-java/blob/master/src/main/java/org/apache/xml/security/encryption/XMLCipher.java